tests/static/embedded_iframe.html

```diff
@@ -0,0 +1,7 @@
+<html>
+    <head></head>
+    <body>
+        <div id='doneDiv'></div>
+        <iframe src='https://www.youtube.com/embed/gwO7k5RTE54?wmode=opaque&amp;enablejsapi=1' onload='document.getElementById("doneDiv").innerText = "Done!"'></iframe>
```

Comment:
Maybe let's not use `youtube.com` here, but something we control, like test.k6.io?

Reply:
I tried that earlier, but it didn't fail :/ I'm not sure what youtube.com does that forces the locel -> remote frame transition.